assets/dev/scss/frontend/widgets/alert.scss

```diff
@@ -18,15 +18,31 @@
 
 	button.elementor-alert-dismiss {
 		position: absolute;
-		@include end(10px);
-		top: 10px;
+		@include end(var(--dismiss-icon-horizontal-position, 10px));
+		top: var(--dismiss-icon-vertical-position, 10px);
 		padding: 3px;
-		font-size: 20px;
+		font-size: var(--dismiss-icon-size, 20px);
 		line-height: 1;
 		background: transparent;
-		color: inherit;
+		color: var(--dismiss-icon-normal-color, inherit);
 		border: none;
 		cursor: pointer;
+	  	transition-duration: var(--dismiss-icon-hover-transition-duration, 0.3s);
+
+		&:hover {
+		  	color: var(--dismiss-icon-hover-color, inherit);
+		}
+
+	  	svg {
+			width: var(--dismiss-icon-size, 20px);
+			height: auto;
+			fill: var(--dismiss-icon-normal-color, currentColor);
```

Comment:
Clarfication question: where/how is currentColor defined? I have never seen it before.

Reply:
@hein-obox If you want a Googled definition:

The currentColor keyword represents the value of an element's color property. This lets you use the color value on properties that do not receive it by default. If currentColor is used as the value of the color property, it instead takes its value from the inherited value of the color property.

If I remember correctly I basically needed to use `currentColor` because the SVG fill defaults to `#000000` if nothing is specified and we wanted it to inherit the parent element's color.

Reply:
@louiswol94 Exactly! Good one!